lib/internal/Magento/Framework/Logger/Test/Unit/Handler/SystemTest.php

```diff
@@ -79,15 +83,21 @@ public function testWriteException()
      * @param array $context
      * @return array
      */
-    private function getRecord($level = Logger::WARNING, $message = 'test', $context = [])
-    {
+    private function getRecord(
+        int $level = Logger::WARNING,
+        string $message = 'test',
+        array $context = []
+    ):array {
```

Comment:
```suggestion
    ): array {
```

Reply:
Hello @ihor-sviziev 
Thank you for your notice.
I fixed it.